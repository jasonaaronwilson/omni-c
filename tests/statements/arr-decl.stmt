char buf[4096];
