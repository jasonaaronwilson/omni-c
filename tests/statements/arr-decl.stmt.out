{
    tag = PARSE_NODE_VARIABLE_DEFINITION
    name = buf
    type = {
        tag = PARSE_NODE_TYPE
        type_node_kind = TYPE_NODE_KIND_PRIMITIVE_TYPENAME
        type_name = char
        type_args = [
        ]
    }
    suffixes = [
        {
            tag = PARSE_NODE_BALANCED_CONSTRUCT
            start_token = "["
            end_token = "]"
        }
    ]
}

// C Output
char buf[4096];

